lib/minitest/mock.rb

```diff
@@ -222,8 +227,10 @@ def stub name, val_or_callable, *block_args
 
     yield self
   ensure
-    metaclass.send :undef_method, name
-    metaclass.send :alias_method, name, new_name
-    metaclass.send :undef_method, new_name
+    unless metaclass.nil?
```

Comment:
It seems a bit odd to have a conditional in an `ensure` block, but at least this is better than depending on the `metaclass` being defined or not.